docs/UserGuide.md

```diff
@@ -83,13 +90,20 @@ Any other arguments entered with -view will be ignored.
 ### Other option:
 1. Clone this repository (or [download as zip](https://github.com/reposense/RepoSense/archive/master.zip))
 1. Execute the following command on the OS terminal inside the project directory. <br/>
-Usage: `gradlew run -Dargs="(-config ./configs/ | -repos https://github.com/reposense/RepoSense.git https://github.com/se-edu/collate.git | -view report_path/) [-output OUTPUT_DIRECTORY] [-since DD/MM/YYYY] [-until DD/MM/YYYY] [-formats FORMAT...]"` <br/>
+Usage: `gradlew run -Dargs="([-config ./configs/...] | -repos https://github.com/reposense/RepoSense.git https://github.com/se-edu/collate.git | -view report_path/) [-output OUTPUT_DIRECTORY] [-since DD/MM/YYYY] [-until DD/MM/YYYY] [-formats FORMAT...]"` <br/>
```

Comment:
Since you're editing this line, please help to make a correction. 
Usage suppose to demo using an arbitrary CONSTANT.

Would you please help me change:
`./configs/...` -> `CONFIG_FOLDER`
`https://github.com/reposense/RepoSense.git https://github.com/se-edu/collate.git` -> `REPO_PATH_OR_URL...`
`report_path/` -> `REPORT_FOLDER`